assets/src/scripts/services/state/water-levels.spec.js

```diff
@@ -2,12 +2,12 @@ import { combineReducers, createStore } from 'redux';
 import configureStore from 'redux-mock-store';
 import { default as thunk } from 'redux-thunk';
 
-import cache from './cache';
-import { getWaterLevels, retrieveWaterLevels, setWaterLevels, WATER_LEVELS_SET } from './cache';
+import cache from './water-levels';
```

Comment:
Could `cache` be imported with the stuff in the line below?

Reply:
Yes, we could do: ```import { default as waterLevels } from './water-levels';```